packages/flutter/test/material/expansion_tile_test.dart

```diff
@@ -1621,4 +1621,41 @@ void main() {
     final Offset offsetPlatform = tester.getTopLeft(platform);
     expect(offsetPlatform, const Offset(16.0, 17.0));
   });
+  testWidgets('ExpansionTile uses childrenBackgroundColor Color',
```

Comment:
nit: might be good to have an empty line between tests:) Here and the test below.